src/MarketAPI.php

```diff
@@ -45,7 +45,7 @@ public function validateRequest(array $request): bool
     /**
      * @throws \Exception
      */
-    protected function makeRequest(string $method, array $request): array
+    public function makeRequest(string $method, array $request): array
```

Comment:
i think it must be protected or private

Reply:
Works fine for me when it's set to public, but doesn't hurt if it's set to protected I guess.